lib/puppetlabs_spec_helper/tasks/fixtures.rb

```diff
@@ -1,409 +1,414 @@
 # frozen_string_literal: true
 
+# rubocop:disable Layout/ClassAndModuleChildren
+
 require 'yaml'
 require 'open3'
 require 'json'
 
-# Top level namespace for spec helper tasks.
-module PuppetlabsSpecHelper::Tasks end
-
 # Helpers for workfing with fixtures.
-module PuppetlabsSpecHelper::Tasks::FixtureHelpers
-  # This is a helper for the self-symlink entry of fixtures.yml
-  def source_dir
-    Dir.pwd
-  end
-
-  # @return [String] - the name of current module
-  def module_name
-    raise ArgumentError unless File.file?('metadata.json') && File.readable?('metadata.json')
-
-    metadata = JSON.parse(File.read('metadata.json'))
-    metadata_name = metadata.fetch('name', nil) || ''
-
-    raise ArgumentError if metadata_name.empty?
+module PuppetlabsSpecHelper
+  # The Tasks module.
```

Comment:
This comment feels a bit redundant.

Reply:
Another rubocopism.

I might actually spend a few minutes and see if there is a nicer way to do this.

Reply:
Yes, technically you can document modules. If you use rdoc or yardoc it will create some documentation. However, I always find no documentation better than some placeholder to please a lint check. You can disable the cop and I'd recommend that.